TestAssets/NonRestoredTestProjects/TestProjectWithUnresolvedPlatformDependency/project.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "ThisIsNotARealDependencyAndIfSomeoneGoesAndAddsAProjectWithThisNameIWillFindThemAndPunishThem": {
```

Comment:
This name was taken from an existing test asset testing a "project" dependency: https://github.com/dotnet/cli/blob/rel/1.0.0/TestAssets/NonRestoredTestProjects/TestProjectWithUnresolvedDependency/project.json#L11